config/runtime.exs

```diff
@@ -62,12 +62,24 @@ github_app_id =
 github_app_name =
   System.get_env("GITHUB_APP_NAME") ||
     Application.get_env(:lightning, :github_app, [])
-    |> Keyword.get(:app_id, nil)
+    |> Keyword.get(:app_name, nil)
+
+github_app_client_id =
+  System.get_env("GITHUB_APP_CLIENT_ID") ||
+    Application.get_env(:lightning, :github_app, [])
+    |> Keyword.get(:client_id, nil)
+
+github_app_client_secret =
+  System.get_env("GITHUB_APP_CLIENT_SECRET") ||
+    Application.get_env(:lightning, :github_app, [])
+    |> Keyword.get(:client_secret, nil)
```

Comment:
If mandatory on prod, would move to prod section and use a `Keyword.fetch!`. Unless we have some instances where the Github feature would be turned off (possibly on a cluster or set of nodes for free users). cc @taylordowns2000

Reply:
I have talked with Taylor and the code is correct. The feature might not be used on prod on certain instances.